lab-cortney/controller/auth-controller.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// auth-controller takes a req.body containing a new username and password,
+// utilizes the User model to ensure unique username and hash,
+// then sends user a token of... authenticity? for what?
```

Comment:
When you sign up, there are a few or more ways to do it. 
1.   User types his/her new username and password into their signup form --> we hash the password --> store hashed password & username in our DB (never save non-hashed password) --> then, the user needs to go to the login page to have access to the app/site because login page will give you back token as well if you are not proving that in signup.

2. User types his/her new username and password into their signup form --> we hash the password --> store hashed password & username in our DB (never save non-hashed password) --> then, give user/client side back a token so that they can have access to the app/site without going through the login page.